src/Shared/XMakeAttributes.cs

```diff
@@ -76,6 +77,7 @@ internal struct MSBuildRuntimeValues
             internal const string clr2 = "CLR2";
             internal const string clr4 = "CLR4";
             internal const string currentRuntime = "CurrentRuntime";
+            internal const string net = "NET";
```

Comment:
@baronfel I'd like your input on this new name please. We need to add an entry to the list that indicates ".NET Core or .NET 5.0+" and is distinguishable from the longstanding `CLR2` and `CLR4`. We should probably get buyoff from somebody in .NET Runtime PM about it.

Reply:
Reaching out to some folks for discussion :thumbsup:

Reply:
Where are these values exposed? Presumably you want input on the literal itself, not the name of the constant field, right?

Reply:
I lean towards "NET" now, personally. There's a long internal email thread where I think we've settled on this nomenclature.